Draft version of patch, that makes possible to define SHARED_DATA_DIR as
absolute path.

Notes: possibly breaks loading of data from user's home folder

--- source/vdrift/pathmanager.cpp.orig	2013-04-22 14:44:09.626544667 +0400
+++ source/vdrift/pathmanager.cpp	2013-04-22 14:45:01.010546299 +0400
@@ -176,7 +176,7 @@
 		//dirs.push_back(user_data_dir);
 
 		// Adding relative path from installed executable
-		dirs.push_back(execname().parent_path().parent_path() / shareDir);
+		dirs.push_back(shareDir);
 		// Adding relative path for running from sources
 		dirs.push_back(execname().parent_path().parent_path() / "data");
 		dirs.push_back(execname().parent_path().parent_path());
